handler/step.go

```diff
@@ -66,7 +67,7 @@ func HandleStartStep(config *config.Config) http.HandlerFunc {
 				v.Path = hv.HostPath.Path
 			}
 		}
-
+		updateDelegateCapacity(&s.StartStepRequestConfig)
```

Comment:
wouldn't it be better to add this to `HandleSetup` instead of `HandleStartStep` since that's when the init task comes?

Reply:
Delegate information is not available at that time, In CI Manager. We know the delegate only when pod has initialised.